packs/vsphere/actions/workflows/vm_hw_basic_build.yaml

```diff
@@ -38,12 +39,13 @@ vsphere.vm_hw_basic_build:
         description: <% $.vm_description %>
         vsphere: <% $.vsphere %>
       publish:
-        vm_id: <% $.barebones_build.result.vm_id %>
+        vm_id: <% task(barebones_build).result.result.vm_id %>
       on-success:
         - add_controller
     add_controller:
       # [186, 198]
       action: vsphere.vm_hw_scsi_controller_add
+      wait-after: 12
```

Comment:
Just curious - why are all those wait-after needed?

Are the operations async and we need to wait a bit before proceed with other operations? Or it it something else?